### PR TITLE
Fix bug in getShowkaseMetadataFromPreview where the preview width was being set as the showkase metadata height

### PR DESCRIPTION
**Pull request by stefankram**
This PR contains a fix for an issue I discovered when trying to get the height information of a `@Preview` from a `ShowkaseBrowserComponent`.

Please let me know if I need to add any other tests to get this merged.

### CODE REVIEW

File: showkase-processor/src/main/java/com/airbnb/android/showkase/processor/models/ShowkaseMetadata.kt [hunk 1]
```diff
@@ -287,7 +287,7 @@ internal fun getShowkaseMetadataFromPreview(
             showkaseName = showkaseName,
             showkaseGroup = showkaseGroup,
             showkaseWidthDp = if (width == -1) null else width,
-            showkaseHeightDp = if (height == -1) null else width,
```
Comment (vinaygaba) [inline showkase-processor/src/main/java/com/airbnb/android/showkase/processor/models/ShowkaseMetadata.kt:290]: oh great catch!

**Comment by vinaygaba** [comment]
@stefankram seems like some tests are failing. Will you do a pass and fix them 🙏

**Comment by stefankram** [comment]
@vinaygaba yep will take a look!

**Comment by stefankram** [comment]
@vinaygaba this should be good to re-test

**Comment by stefankram** [comment]
Hey @vinaygaba I noticed a new version was created. I was hoping to get this PR merged and into that release...

Is there anything else I can help with to get this merged down?

**Comment by vinaygaba** [comment]
@stefankram about to merge this! Just re-running the build

**Review by vinaygaba — APPROVED** [review]

**Pull request merged**